mcmahon.py

```diff
@@ -215,7 +210,7 @@ def generate_pairing(self, sample_size):
     def add_result(self, round_, board, winner):
         match = self.rounds[round_][board]
         match.winner = winner
-        self.players[winner].mm_score += 1
+        self.players[winner].mm_score[0] += 1
```

Comment:
At this point, is there even any reason to do this? It leaves SOS and SODOS incorrect, and we basically assume we are going to manually trigger a recomputation down the line.